raytracing/gaussianbeam.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, q: complex = None, w: float = None, R: float = float("+Inf"),
             raise ValueError("Please specify 'q' or 'w'.")
 
         if q is not None and w is not None:
-            if not cmath.isclose(a=self.q, b=q, rel_tol=relTol):
+            if areNotEqual(self.q.real, q.real) or areNotEqual(self.q.imag, q.imag):
```

Comment:
Is it possible to have a *relative* difference instead of an absolute one? If real values or imag values are huge, an absolute difference of, let's say epsilon = 1e- 3 doesn't make sense.

Reply:
Yes but we will need to be careful : relative when the value is near 0 cannot be done easily.
We should create `areAbsolutelyNotEqual()` and `areRelativelyNotEqual()` for instance.

Reply:
I created relative methods, but I'm not sure if they are robust enough when close to 0.